src/base/rss/rssarticle.h

```diff
@@ -69,6 +69,8 @@ namespace Rss
         QVariantHash toHash() const;
         static ArticlePtr fromHash(Feed *parent, const QVariantHash &hash);
 
+        const QString getValue(const QString &name);
```

Comment:
```c++
QString getValue(const QString &name) const;
```

Any non-modifier methods should be defined as const-methods (ie with const at the end) so we'll can call it with const objects.